reactor-netty-http/src/test/java/reactor/netty/http/server/HttpServerTests.java

```diff
@@ -836,6 +769,9 @@ void contextShouldBeTransferredFromDownStreamToUpStream() {
 			                  .get("Hello")
 			                  .equals("World")).isTrue();
 		}
+
+		provider.disposeLater()
```

Comment:
in case of exception, the `provider` wouldn't be disposed.

Reply:
fixed